tests/testing_utils.py

```diff
@@ -95,3 +96,18 @@ def full_path(self, filename):
         f.full_path('foo/bar.txt') -> /tmp/asdfasd/foo/bar.txt
         """
         return os.path.join(self.rootdir, filename)
+
+
+class TempFile(object):
```

Comment:
You can use `with tempfile.NamedTemporaryFile(delete=False)` instead. This is how all of our other tests are setup, so let's keep everything consistent.

Reply:
thanks for the hint. Didn't know there is this method. 

Why we are going with `delete=False`?